unitTests/ui/BreadcrumbValuesListTest.ts

```diff
@@ -131,6 +131,14 @@ export function BreadcrumbValueListTest() {
         expect($$(captions[1]).text()).toContain('foo1');
         expect($$(captions[2]).text()).toContain('foo2');
       });
+
+      it('should display a label with the right attributes for each excluded values', () => {
+        const built = new BreadcrumbValueList(facet, facetValues, BreadcrumbValueElement).build();
+        const values = $$(built).findAll('.coveo-facet-breadcrumb-value')[2];
+        expect($$(values).getAttribute('aria-label')).toBe('Remove exclusion filter on foo2');
+        expect($$(values).getAttribute('role')).toBe('button');
+        expect($$(values).getAttribute('tabindex')).toBe('0');
```

Comment:
Not sure about that test. I didn't see any test for the aria-label when a facet is excluded. Thought I might add one.

Reply:
Thats nice, you can rename the test like the others:
`has the right accessibility attributes`